docs/auth/Single Sign-On via Device Browser.adoc

```diff
@@ -0,0 +1,112 @@
+```
+manifestPlaceholders = ['appAuthRedirectScheme': 'org.aerogear.mobile.emailapp']
+```
+Where `org.aerogear.mobile.emailapp` is your package identifier. +
+
+. Now you need to instruct the Auth SDK in the Email App to use this redirect uri. To do this the Auth SDK needs to be initialised.  A quick guide on how to initialise the Auth SDK can be found here - link:/toAidensDocs.adoc[Initializing the SDK].  +
+[NOTE]
+this is the redirect uri that needs to be added to the *Keycloak Email Client* i.e. one of the *Valid Redirect URIs* should be `org.aerogear.mobile.emailapp:/callback`. +
+
+
+. Lastly, your app needs a `mobile-service.json` configuration file. In this file you can specifiy different services your app uses like Keycloak.  So the information entered previously for <<Setup Keycloak Email Client>> will need to be specified in this JSON file.  The configuration should have the following configuration items:
+```
+{
+    "version": 1,
+    "clusterName": "https://192.168.37.1:8443",
+    "namespace": "myproject",
+    "services": [
+        {
+            "id": "keycloak",
+            "name": "keycloak",
+            "type": "keycloak",
+            "url": "https://keycloak-myproject.192.168.37.1.nip.io",
+            "config": {}
+        }
+    ]
+}
+```
+*`clusterName`*: This is the URL to your Openshift cluster. +
+*`namespace`*: This is the project in Openshift where you can access resources e.g. the two Android app's you have created. +
+*`services`*: These are the services your app uses.  For this setup we are only using Keycloak. +
+*`id`*: The ID of the service. +
+*`name`*: The name of the service. +
+*`type`*: The type of the service. +
+*`url`*: This is your Keycloak server base url. +
+*`config`*: The configuration for the service specified. This configuration is for the Email App so it can be got by navigating to the Clients section in Keycloak, clicking clients and navigating to the *Installation* tab.  In this tab you can select *Keycloak OIDC JSON* from the *Format Option* dropdown and copy and paste that configuration into the config object here. +
+So after adding in that Keycloak Email Client configuration, the `mobile-service.json` file will look like: 
+
+```
+{
+    "version": 1,
+    "clusterName": "https://192.168.37.1:8443",
+    "namespace": "myproject",
+    "services": [
+        {
+            "id": "keycloak",
+            "name": "keycloak",
+            "type": "keycloak",
+            "url": "https://keycloak-myproject.192.168.37.1.nip.io",
+            "config": {
+                  "realm": "myproject",
+                  "auth-server-url": "https://keycloak-myproject.192.168.37.1.nip.io/auth",
+                  "ssl-required": "external",
+                  "resource": "emailapp",
+                  "public-client": true,
+                  "use-resource-role-mappings": true,
+                  "confidential-port": 0
+            }
+        }
+    ]
+}
+```
```

Comment:
Given the context this doc is for, I think we should mention that the file can be generated by the mobile-cli tool. In this case, if developer already has this file available locally, they need to check the details of the keycloak settings to make sure they are matching the configuration in KC.

Reply:
Right ok, good catch.  I'll include this information